src/index.js

```diff
@@ -1,6 +1,7 @@
 import {spawn} from 'cross-spawn'
 import commandConvert from './command'
 
+module.exports = crossEnv
 export default crossEnv
```

Comment:
Either way is fine, I'll do a squash and merge when I merge your pull request anyway